content/docs/user-guide/how-to/setup-google-drive-remote.md

```diff
@@ -269,8 +269,8 @@ heavy usage, it is recommended to rely on
 </admon>
 
 > This requires having your own
-> [GC project](/doc/user-guide/how-to/setup-google-drive-remote#using-a-custom-google-cloud-project-recommended)
-> as explained above.
+> [GC project](#using-a-custom-google-cloud-project-recommended) as explained
+> above.
```

Comment:
Finally, just a quick internal link fix here for now, but this page ([GDrive](https://dvc.org/doc/user-guide/how-to/setup-google-drive-remote#how-to-setup-a-google-drive-dvc-remote)) would probably be next (turn it into a regular remote config page). Although maybe Aliyun OSS is easier (similar to the ones that are done)... 🤔